assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -288,6 +306,9 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
         setTransformStyle(target);
       }}
       onRotateEnd={({ target }) => {
+        if (wasElementDeleted(target)) {
+          return;
+        }
```

Comment:
Similarly here. I would not expect this to happen nor use it as a method to delete.